src/SecurityManager.h

```diff
@@ -0,0 +1,112 @@
+#ifndef SecurityManager_h
+#define SecurityManager_h
+
+#include <list>
+#include <ArduinoJsonJWT.h>
+#include <ESPAsyncWebServer.h>
+
+#define DEFAULT_JWT_SECRET "esp8266-react"
+
+#define AUTHORIZATION_HEADER "Authorization"
+#define AUTHORIZATION_HEADER_PREFIX "Bearer "
+#define AUTHORIZATION_HEADER_PREFIX_LEN 7
```

Comment:
Consider 
```suggestion
#define AUTHORIZATION_HEADER_PREFIX_LEN sizeof(AUTHORIZATION_HEADER_PREFIX)
```
so the value and length can't get out of sync if ever updated.

Reply:
Might this return 8 rather than 7, is there an implicit null terminator?

Reply:
Ooh, good catch. `sizeof()` includes the nul terminator; `strlen()` doesn't.

Reply:
Will the compiler factor this out to a static constant int at build time? I'm worried it will create a string and evaluate it every time the define is processed..

Reply:
My unfamiliarity with C coming through here...

Reply:
Sorted.